smart-contract-verifier/smart-contract-verifier-server/src/types/sourcify.rs

```diff
@@ -0,0 +1,76 @@
+use serde::{Deserialize, Serialize};
+use smart_contract_verifier::sourcify::api::VerificationRequest;
+use smart_contract_verifier_proto::blockscout::smart_contract_verifier::v1::VerifyViaSourcifyRequest;
+use std::ops::Deref;
+
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+pub struct VerifyViaSourcifyRequestWrapper(VerifyViaSourcifyRequest);
```

Comment:
couldn't we just make a `fn convert_sourcify(VerifyViaSourcifyRequest) -> VerificationRequest` and have like 10% of the code? :)

Reply:
We canRead https://github.com/blockscout/blockscout-rs/pull/224#discussion_r1034041322 for more details